src/package/nuspec/Microsoft.CodeCoverage.nuspec

```diff
@@ -36,6 +36,12 @@
     <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Core.dll" target="build\netstandard1.0\" />
     <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Interprocess.dll" target="build\netstandard1.0\" />
     <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Instrumentation.dll" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\Microsoft.CodeCoverage.Telemetry.dll" target="build\netstandard1.0\" />
```

Comment:
how big are those dlls? I'm wondering if it is good to add it also to .netstandard version of collector.

Reply:
0.98MB combined